src/platform/mhv/secondary-nav/tests/MhvSecondaryNav.unit.spec.jsx

```diff
@@ -92,4 +100,36 @@ describe('MHV Secondary Navigation Menu Component', () => {
       cleanup(); // must be done after the render
     });
   });
+
+  describe('transitional Medical Records page -- enabled', () => {
+    it('renders the /my-health/records link', async () => {
+      const store = mockStore({
+        mhvSecondaryNavigationEnabled: true,
+        mhvTransitionalMedicalRecordsLandingPage: true,
+      });
+      const { findByRole } = render(
+        <Provider store={store}>
+          <MhvSecondaryNav items={testSecNavItems} />
+        </Provider>,
+      );
+      const result = await findByRole('link', { name: /Records$/ });
+      expect(result.getAttribute('href')).to.eq('/my-health/records');
+    });
+  });
+
+  describe('transitional Medical Records page -- disabled', () => {
```

Comment:
Perhaps remove this describe, so the two tests are grouped. That will make the describe above just `transitional Medical Records page` and the tests inside enabled and disabled.

Reply:
When we remove the feature toggle, we can easily identify this block of code and delete it.